cloudinit/sources/DataSourceOracle.py

```diff
@@ -192,13 +127,20 @@ def _get_data(self):
 
         # network may be configured if iscsi root.  If that is the case
         # then read_initramfs_config will return non-None.
-        if _is_iscsi_root():
-            data = read_opc_metadata()
-        else:
-            with dhcp.EphemeralDHCPv4(net.find_fallback_nic()):
-                data = read_opc_metadata()
+        fetch_vnics_data = self.ds_cfg.get('configure_secondary_nics', False)
+        # suppress() in this context is just a no-op context manager
+        network_context = suppress() if _is_iscsi_root(
```

Comment:
I don't love using suppress here, but duplicating the `read_opc_metadata()` call (especially if it's taking arguments now) seems worse to me.